tools/build/src/mocha-reporter/index.ts

```diff
@@ -76,6 +97,17 @@ class BentleyMochaReporter extends Spec {
           const wtf = require("wtfnode");
           wtf.setLogger("info", console.error);
           wtf.dump();
+
+          let activeResourcesInfo: string[] = (process as any).getActiveResourcesInfo(); // https://nodejs.dev/en/api/v18/process#processgetactiveresourcesinfo (Not added to @types/node yet I suppose)
```

Comment:
youve tried updating to the latest @types/node@18.x ver?

Reply:
No I didn't, but I just checked definitelyTyped repo and I don't see it mentioned anywhere except for a couple of comments in async_hooks types file.